src/lib/viewers/image/ImageControls.tsx

```diff
@@ -26,6 +32,7 @@ export default function ImageControls({
             <RotateControl onRotateLeft={onRotateLeft} />
             <FullscreenToggle onFullscreenToggle={onFullscreenToggle} />
             <AnnotationsControls
+                annotationColor={annotationColor}
```

Comment:
I haven't added the business logic yet for ImageControls (as well as some of the UI for showing the ColorPickerControl for an image file). Would it be okay for me to include that within this PR as well?